frontend/src/sass/partials/_helpers.scss

```diff
@@ -14,6 +14,42 @@
 	margin-top: 15px !important;
 }
 
+.margin-bottom-15 {
+    margin-bottom: 15px !important;
+}
+
+.margin-top-30 {
+    margin-top: 30px !important;
+}
+
+.margin-bottom-30 {
+    margin-bottom: 30px !important;
+}
+
+.text-align-center {
+    text-align: center !important;
+}
+
+.text-align-left {
+    text-align: left !important;
+}
+
+.text-align-right {
+    text-align: right !important;
+}
+
+.background--primary {
+    background-color: $pastel-green;
+}
+
+.unstyled-list {
+    margin: 0 !important;
+    padding: 0 !important;
+
+    li {
+        list-style: none !important;
+    }
+}
```

Comment:
What is the reason of so extensive use of `!important` rule here?

Reply:
as these are helpers they need to trump all other styles, so here and if we had base styles would be the only place to do so that is actually appropriate